scripts/docker-compose-openfn.yml

```diff
@@ -5,16 +5,16 @@ networks:
 services:
   postgresql:
     env_file:
-      - '${MSF_ENV_CONFIG_PATH}/.env'
-    stop_grace_period: '3s'
+      - "${MSF_ENV_CONFIG_PATH}/.env"
+    stop_grace_period: "3s"
     volumes:
       - "${SQL_SCRIPTS_PATH}/postgresql/create_openfn_db.sh:/docker-entrypoint-initdb.d/create_openfn_db.sh"
   migrations:
-    platform: linux/x86_64/v8
-    image: 'openfn/lightning:v2.11.0'
```

Comment:
@jnsereko do we need these changes committed?

Reply:
no